MatchCalculator.java

```diff
@@ -0,0 +1,43 @@
+package UnitOmega13;
+
+public class MatchCalculator {
```

Comment:
you must use ternary operator in this task
https://www.baeldung.com/java-ternary-operator

Reply:
you can make this task writing only one method